data/boundaries/citycountystate/index.html

```diff
@@ -165,8 +152,7 @@ <h5 id="CountyBoundaries-downloads">Downloads</h5>
     </div>
     <div class="panel-content">
       <ul class="dotless no-padding" id="CountyBoundaries-dlist">
-        <li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mbnFpYUl1bDZtaDQ&export=download">County Boundaries: File Geodatabase</a></li>
-        <li><a href="https://drive.google.com/a/utah.gov/uc?id=0ByStJjVZ7c7mc3BGRXhKa0N0U2c&export=download">County Boundaries: Shapefile</a></li>
+        <li><a href="https://opendata.gis.utah.gov/datasets/utah-county-boundaries">County Boundaries: SGID Open Data Downloads and Services</a>
```

Comment:
Don't we have a section for web services on the data pages as well? I wonder if this link should go there? Maybe in both download and web services? Just a thought.

Reply:
yeah i thought that was redundant but am up for a mind change. I think the data pages could actually use a refresh with our new sgid. Let's chat about it at the next status meeting.